python/smqtk/algorithms/saliency/logit_sal.py

```diff
@@ -0,0 +1,247 @@
+from PIL import Image
+import numpy as np
+import os
+from sklearn.metrics.pairwise import euclidean_distances
+import six
+from smqtk.algorithms.relevancy_index import get_relevancy_index_impls
+from smqtk.algorithms import get_descriptor_generator_impls
+from smqtk.algorithms import SmqtkAlgorithm
+from smqtk.algorithms.saliency import SaliencyBlackbox,ImageSaliencyAugmenter
+from smqtk.representation.data_element.memory_element import DataMemoryElement
+from smqtk.utils import plugin
+
+    
+class Logit_SaliencyBlackbox (SaliencyBlackbox):
+    """
+    Logit_SaliencyBlackbox function yields some floating point scalar value for a given masked base image descriptor element that signifies the difference between the confidence value of the query image and masked image descriptors, used by class implementations of 'ImageSaliencyMapGenerator'.
+    """
+
+    def __init__(self, ADJs, rel_index, T_descr):
+        """
+        :param ADJs: is a set of positive and negative adjudications.
+        :param type: set of (smqtk.representation.DescriptorMemoryElemets)
+        :param rel_index: Plugin implementation of the algorithms used to generate
+        relevance index used to rank images
+        :param type: A new instance of a class implementing the
```

Comment:
Same type docstring comments above. I'm just going to assume this update is needed throughout and will not comment on this in every instance.